### DISCUSSION TITLE
ReInit output audio device

### DISCUSSION THREAD

**Discussion by wrun** [sipsorcery-org/sipsorcery#464] [Q&A]
How I can to reinit output audio device (I use USB, and sometimes it`s lost for few seconds).
Or I want to change output audio device in real time. 
Us I understand you use NAudio, but it private in your project. Maybe I must to make my own VoIPMediaSession, but how...

**Comment by sipsorcery** [comment]
See [SIPSorceryMedia.Windows](https://github.com/sipsorcery-org/SIPSorceryMedia.Windows) for the source code that integrates with NAudio. I don't have any experience re-initialising an audio device with NAudio.
